pylib/aeon/eos/connector.py

```diff
@@ -71,7 +71,8 @@ def configure(self, contents):
         if not isinstance(contents, list):
             raise RuntimeError('contents must be a list, for now')
 
-        contents = ['enable', 'configure'] + filter(bool, contents)   # filters out empty lines
+        # filters out empty lines
+        contents = ['enable', 'configure'] + list(filter(bool, contents))
```

Comment:
Filter always returns a list right, why make it a list again ?

Reply:
can be a filter object depend on python version